javascript/apps/taiga/src/app/modules/project/story-detail/story-detail.component.html

```diff
@@ -258,6 +258,7 @@
 
                   <tg-comments
                     [comments]="vm.comments"
+                    [story]="vm.story"
```

Comment:
tg-comments should not use story, because in future stories we are going to have other objects that have comments. Maybe you can use an output and do what you need in story-detail.